### ISSUE TITLE
revise hackage bounds

### ISSUE THREAD

**Issue by vmchale** [timbod7/haskell-chart#255]
Right now `Chart-chairo` 1.9.4 fails to build with:

```
[1 of 1] Compiling Graphics.Rendering.Chart.Backend.Cairo ( Graphics/Rendering/Chart/Backend/Cairo.hs, /tmp/Chart-cairo-1.9.4/dist-newstyle/build/x86_64-linux/ghc-9.6.3/Chart-cairo-1.9.4/build/Graphics/Rendering/Chart/Backend/Cairo.o, /tmp/Chart-cairo-1.9.4/dist-newstyle/build/x86_64-linux/ghc-9.6.3/Chart-cairo-1.9.4/build/Graphics/Rendering/Chart/Backend/Cairo.dyn_o )

Graphics/Rendering/Chart/Backend/Cairo.hs:192:21: error: [GHC-88464]
    Variable not in scope: void :: IO (PickFn ()) -> IO ()
    |
192 | toFile fo path ec = void $ renderableToFile fo path (toRenderable (execEC ec))
    |                     ^^^^
cError: cabal: Failed to build Chart-cairo-1.9.4.
```

Thanks!

**Comment by georgefst** [comment]
Similar for `Chart-diagrams` (`void` and `forM_`).

**Comment by bravit** [comment]
Thank you! I've revised the bounds on hackage and published Chart-cairo 1.9.4.1 and Chart-diagrams 1.9.5.1 which are acually buildable with mtl-2.3.
